### PR TITLE
Add UI for calculator

### PR DESCRIPTION
**Pull request by bizip**
**What does this PR do?**

- Add UI for calculator

**Description of Task to be completed**

All tasks related to deployment are completed 

**How should this be manually tested?**

1. Clone this repository and switch to `feature-startup`
2. Install all required dependencies `npm install`
3. Run `npm start`

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,9 +1,10 @@
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
-    <div className="App">
-      <h1>Math magicians</h1>
+    <div className="app">
+      <Calculator />
     </div>
```
Comment (Wes-Isaac) [inline src/App.js]: - [ ] Great job so far, to be consistent it's better to use a class-based component, kindly fix that.

File: src/components/Calculator.module.css [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  grid-gap: 0.3rem;
+  background-color: rgb(185, 182, 182);
+}
+
+.calc__btn {
+  background-color: silver;
+  height: 10rem;
+}
```
Comment (Wes-Isaac) [inline src/components/Calculator.module.css]: - [ ] [_OPTIONAL_] It would be better to access the calculator without scrolling so that it's easy to use.

**Review by Wes-Isaac — CHANGES_REQUESTED** [review]
Hi @bizip, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Linters are set up properly ✔️.
- Great summary😲. 
- Project works as expected 😮

## Required Changes ♻️

- [ ] Kindly remove unused files generated while creating the react app as it's a requirement.
image


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by KDlamini — APPROVED** [review]
### Hi @bizip 🙋‍♀️,  

Great job! 💯 
You have done a great job implementing all the required changes. 🎊 🥳 
Your feature is complete! There is nothing else to say other than... it's time to merge it. :shipit: 
Congratulations!!!

# Status: Approved ✔️ 

simon

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini  in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**